googlemock/include/gmock/gmock-spec-builders.h

```diff
@@ -499,7 +499,8 @@ class GTEST_API_ Expectation {
  public:
   // Constructs a null object that doesn't reference any expectation.
   Expectation();
-
+  Expectation(const Expectation&) = default;
```

Comment:
Are these 2 lines of change related to the removal `GTEST_DISALLOW_ASSIGN_(ExpectationBase)`?
What about the move ctor and assignment operator?

Reply:
These are in a separate commit (look at the "Commits" tab). It fixes a -Wdeprecated warning.
I guess it would make sense to default the move operations too, yeah. Will do.

Reply:
Do you want to add the move ctor and assignment before I import it?

Reply:
Done!